<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakerDialog</class>
 <widget class="QDialog" name="MakerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MakerDialog</string>
  </property>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_step">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>步进</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_init">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>地图初始化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_erosion">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>腐蚀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>210</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_run">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>133</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rdb_fault">
      <property name="text">
       <string>断层构造法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_plate">
      <property name="text">
       <string>板块漂移法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_particle">
      <property name="text">
       <string>粒子沉积法</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rdb_particle_sand">
      <property name="text">
       <string>粒子沉积法（沙子）</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
